types/wlr_drm_lease_v1.c

```diff
@@ -0,0 +1,630 @@
+#include <assert.h>
+#include <errno.h>
+#include <stdlib.h>
+#include <stdio.h>
+#include <unistd.h>
+#include <wlr/backend/drm.h>
+#include <wlr/backend/multi.h>
+#include <wlr/types/wlr_drm_lease_v1.h>
+#include <wlr/util/log.h>
+#include <wayland-util.h>
+#include "backend/drm/drm.h"
```

Comment:
I think we've stopped using private DRM backend functions?

Reply:
I'm using `get_drm_connector_from_output` and `get_drm_backend_from_backend`